docs/plugins/inputs/kafka.asciidoc

```diff
@@ -132,6 +149,14 @@ See the https://kafka.apache.org/{kafka_client_doc}/documentation for more detai
 | <<plugins-{type}s-{plugin}-request_timeout_ms>> |<<number,number>>|No
 | <<plugins-{type}s-{plugin}-retry_backoff_ms>> |<<number,number>>|No
 | <<plugins-{type}s-{plugin}-sasl_client_callback_handler_class>> |<<string,string>>|No
+| <<plugins-{type}s-{plugin}-sasl_oauthbearer_token_endpoint_url>> |<<string,string>>|No
```

Comment:
Settings (list and description) should be in alphabetical order. 
I opened https://github.com/logstash-plugins/logstash-integration-kafka/issues/196 to address the issue in a future PR.